INTEGRATION.md

```diff
@@ -44,34 +77,37 @@ The below decision tree defines the decisions to be made in order to choose the
       A[Has gateway argument] --> |yes| B(gateway from argument);
       A[Has gateway argument] --> |no| C(try IPFS_GATEWAY env);
```

Comment:
"argument" feels focused on command-line use case, but we may have a GUI application that has custom  gateway setting in own preferences, and no argument per se  (like Brave).

Maybe "Has own gateway config" → "Gateway from own config" ?